ParkingZone/Areas/Identity/Data/ParkingZoneDbContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore;
+using ParkingZone.Areas.Identity.Data;
+
+namespace ParkingZone.Data;
```

Comment:
Data folder should be separate from other folders, so move this Data folder outside of Areas. and name it AppDbContext or ApplicationDbContext because in this context we will store another models like slots, reservations